gsy_framework/validators/heat_pump_validator.py

```diff
@@ -67,18 +67,22 @@ def _validate_temp(cls, **kwargs):
     @classmethod
     def validate_rate(cls, **kwargs):
         """Validate energy rate related arguments."""
-        if not(kwargs.get("initial_buying_rate")
-               and kwargs.get("final_buying_rate")
-               and kwargs.get("update_interval")):
+        if (kwargs.get("initial_buying_rate") is None
+                and kwargs.get("final_buying_rate") is None
+                and kwargs.get("update_interval") is None):
```

Comment:
I like the fix, however I think that this is `or`, not `and`. The original clause says: if not all 3 are not-None. Therefore the converted clause should say: if any of the 3 is None

Reply:
After looking at the following clause though, I think that this clause is redundant, because we raise an exception if any of the 3 is None. I would recommend to remove this completely actually.